app/models/ability.rb

```diff
@@ -97,11 +97,11 @@ def can_administer_provider(persona)
   end
 
   def can_manage_litigator_claims(persona)
-    can_manage_own_claims_of_class(persona, [Claim::LitigatorClaim, Claim::InterimClaim, Claim::TransferClaim])
+    can_manage_own_claims_of_class(persona, Claim::BaseClaim.lgfs_claim_types)
```

Comment:
Delegate the available `agfs/lgfs` claim types to the `Claim` class itself so we don't need to repeat this list all over the code.